src/Client.php

```diff
@@ -16,231 +15,68 @@
  * @method array elevation(string $locations, array $params=[])
  * @method array geocode(string $address, array $params=[])
  * @method array reverseGeocode(string $latlng, array $params=[])
- * @method array computeRoutes(array $origin, array $destination, array $body=[], array $fieldMask=[])
+ * @method array computeRoutes(array $origin, array $destination, array $body=[], array $headers=[], array $params=[])

```

Comment:
That looks wrong

Reply:
\yidas\googleMaps\Services\Routes::computeRoutes - src/Services/Routes.php:59

Reply:
https://github.com/yidas/google-maps-services-php/blob/a6300c722d7516980c882352fdccca9497a1700b/src/Routes.php#L31
My point is that there have been changes in upstream to provide fieldmask directly which seems reverted now in your PR.